xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsMacro.xml

```diff
@@ -373,13 +373,16 @@ require(['deferred!ckeditor', 'xwiki-suggestUsers', 'jquery', 'xwiki-meta'], fun
     </property>
     <property>
       <code>.xwiki-mention {
-  background-color: $services.mentions.mentionsColor;
+  background-color: $services.mentions.mentionsBackground;
```

Comment:
It's using a script service so you should probably also check the Java API and probably makes change there too... That kind of change is not that easy :)

Reply:
I reverted this change in service name. From what I udnerstand, we'd need an API breakage for this, and I'm not sure renaming one property is enough of a reason.